src/EnrollCustomFactorController.js

```diff
@@ -34,21 +34,21 @@ function (Okta, FormController, Footer) {
             factorType: this.get('factorType')
           });
           return factor.enroll()
-          .then((trans) => {
-            setTransaction(trans);
-            var url = this.appState.get('enrollCustomFactorRedirectUrl');
-            if(url !== null) {
-              Util.redirect(url);
-            }
-          })
-          .fail(function (err) {
-            throw err;
-          });
+            .then((trans) => {
```

Comment:
nit: we could add the rule to remove parenthesis on single param arrow functions

Reply:
I think you meant https://eslint.org/docs/rules/arrow-parens
Will enable at next refactor phase if it's preferred.